packages/app-desktop/index.html

```diff
@@ -2,11 +2,19 @@
 <html>
 	<head>
 		<meta charset="UTF-8">
-		<!--
-		No CPS because we need to allow everything due to some dependencies (eg. depd, which comes from maybe Node or Electron
-		uses 'eval'.
-		<meta http-equiv="Content-Security-Policy" content="default-src 'self' 'unsafe-inline' 'unsafe-eval'">
-		-->
+		<meta
+			http-equiv="Content-Security-Policy"
+			content="
+				default-src 'self' joplin-content://* ;
+				connect-src 'self' * http://* https://* joplin-content://* blob: ;
+				style-src 'unsafe-inline' 'self' blob: joplin-content://* https://* http://* ;
+				child-src 'self' joplin-content://* ;
+				script-src 'self' 'unsafe-inline' joplin-content://* ;
+				media-src 'self' * blob: data: https://* http://* joplin-content://* ;
+				img-src 'self' blob: data: http://* https://* joplin-content://* ;
+				font-src 'self' http://* https://* blob: data: joplin-content://* ;
+			"
```

Comment:
The `Content-Security-Policy` added by this pull request doesn't include `unsafe-eval`, but it still includes `unsafe-inline`. Excluding `unsafe-eval` prevents code execution through `eval`, `Function.constructor`, and `setTimeout`. However, it doesn't prevent code execution through HTML `<script>` tags or inline event handlers. Ideally, the desktop app's `Content-Security-Policy` would include neither `unsafe-eval` nor `unsafe-inline`. However, removing `unsafe-inline` is likely to cause issues (particularly with plugins) and may be best done in a follow-up pull request.